lib/webpacker.rb

```diff
@@ -2,6 +2,9 @@
 require "active_support/logger"
 require "active_support/tagged_logging"
 
+ENV["RAILS_ENV"] ||= ENV["RACK_ENV"] || "development"
+ENV["NODE_ENV"] ||= ENV["RAILS_ENV"]
```

Comment:
We probably don't want to set any `ENV` vars here since this file gets required by applications. It was OK to do in bin/* files since those are separate one-off processes.

Reply:
Ahh right, I was thinking setting here will set it in all environments (like in rails console) but I guess that's not ideal.